pkg/virtctl/console/console.go

```diff
@@ -96,7 +94,7 @@ func (c *Console) Run(flags *flag.FlagSet) int {
 	readStop := make(chan error)
 
 	go func() {
-		err := virtCli.VM(namespace).SerialConsole(vm, device, stdinReader, stdoutWriter)
+		err := virtCli.VM(namespace).SerialConsole(vm, stdinReader, stdoutWriter)
```

Comment:
Is there a chance to enable the device selection again?

Reply:
yes, we had that on our apiserver before too.

Reply:
i took the device section out because our api doesn't make the device name configurable right now. If we change the VM api, then we should add the device back.